molgenis-data-security/src/main/java/org/molgenis/data/security/audit/AuthenticationUtils.java

```diff
@@ -25,7 +25,7 @@ static boolean isRunAsSystem() {
         && ((SystemSecurityToken) auth).getOriginalAuthentication().isPresent();
   }
 
-  static String getUsername() {
+  public static String getUsername() {
```

Comment:
There also is a SecurityUtils.getCurrentUsername().
If we make this publicly visible it gets a bit confusing, which should you use use when?